.travis.yml

```diff
@@ -0,0 +1,29 @@
+sudo: required
+
+services:
+  - docker
+
+language: python
+
+python:
+  - "3.9"
+
+install:
+  - pip install pipenv
+  - pipenv install --dev --deploy
+
+script:
+  - PIPENV_PYUP_API_KEY="" pipenv check -i 40291 -i 39253
```

Comment:
Don't think we need `PIPENV_PYUP_API_KEY=""` and `-i 40291 -i 39253` now

Reply:
I checked... we do need it